packages.bzl

```diff
@@ -1,7 +1,7 @@
 # Each individual package uses a placeholder for the version of Angular to ensure they're
 # all in-sync. This map is passed to each ng_package rule to stamp out the appropriate
 # version for the placeholders.
-ANGULAR_PACKAGE_VERSION = "^15.0.0 || ^16.0.0"
+ANGULAR_PACKAGE_VERSION = "^15.0.0 || ^15.1.0-next || ^16.0.0"
```

Comment:
yeah, I think somewhere in Slack we discussed this pre-release range, but it was annoying because it would need to be done for every minor.

Also right now this will break the release tool when we validate the peer range. See tools/release-checks.

Reply:
There are only 2-3 minors between each major so it shouldn't be too bad. I think that this is right range to have here so shouldn't the release tool be updated to reflect it instead?

Reply:
It's a custom check for this repo. See tools/release-checks. Could just update it there. I'm good explicitly allowing the pre-releases (if we account for both minors already).

Reply:
I pushed a separate commit to tweak the prerelase check logic. Maybe I'm being naive, but I think we can get away with something like `^15.0.0 || ^15.1.0-0 || ^16.0.0` only while we're in the prerelase stage of 15.1.0.

Reply:
I think that sound reasonable to me. Just one thing I want to note: Historically we tried to not require the placeholder to change in between majors. This would introduce a little additional churn, but that might just be fine for the components team (just mentioning it here).

Reply:
We can give it a try and see if the caretaker finds it annoying. Personally I think that it should be fine since we have an automated check for it.

Reply:
👍 yeah, that's why we have the checks in the first place